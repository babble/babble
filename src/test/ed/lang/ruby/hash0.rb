
h = { "1" => "one" };
puts( h["1"] );


h = { "1" => "one" , "2" => "two" };
puts( h["1"] );
puts( h["2"] );

h = { 1 => "one" , "2" => "two" };
puts( h[1] );
puts( h["2"] );




