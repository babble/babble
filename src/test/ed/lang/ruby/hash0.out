one
one
two
one
two
